doctr/models/zoo.py

```diff
@@ -39,11 +49,18 @@ def ocr_predictor(
         >>> out = model([input_page])
 
     Args:
-        arch: name of the architecture to use ('db_sar_vgg', 'db_sar_resnet', 'db_crnn_vgg', 'db_crnn_resnet')
+        det_arch: name of the detection architecture to use ('db_resnet50', 'db_mobilenet_v3_large')
```

Comment:
at the beginning of the parenthesis, could you add "e.g." to specify that the list is not exhaustive?